app/controllers/idv/review_controller.rb

```diff
@@ -26,11 +26,23 @@ def new
 
     def create
       resolution = start_idv_session
+      track_idv_event(resolution)
       process_resolution(resolution)
     end
 
     private
 
+    def track_idv_event(resolution)
+      result = {
+        success: resolution.success,
+        idv_attempts_exceeded: idv_attempter.exceeded?,
+        kbv: FeatureManagement.proofing_requires_kbv?,
+        new_phone_added: phone_confirmation_required?
```

Comment:
That sounds right.

The current implementation does this:
> whether they completed IdV after this initial resolution, which may or may not involve a phone confirmation, and may or may not involve KBV.

you are essentially adding coverage for whether initial resolution succeeded or not.